ebos/eclgenericcpgridvanguard.hh

```diff
@@ -28,6 +28,7 @@
 #define EWOMS_ECL_CP_GRID_GENERIC_VANGUARD_HH
 
 #include <ebos/eclgenericvanguard.hh>
+#include <opm/input/eclipse/EclipseState/Grid/LgrCollection.hpp>
```

Comment:
Here, I think we could use a forward  declaration instead
```
class LgrCollection;
```
and move the include to the *.cc file